src/module/canvas/ghost-measured-template.ts

```diff
@@ -32,16 +33,22 @@ export class GhostTemplate extends MeasuredTemplatePF2e {
         if (event.ctrlKey) {
             event.preventDefault();
             event.stopPropagation();
-            const delta = canvas.grid.type > CONST.GRID_TYPES.SQUARE ? 30 : 15;
-            const snap = event.shiftKey ? delta : 5;
-            this.document._source.direction += snap * Math.sign(event.deltaY);
-            this.document.direction += snap * Math.sign(event.deltaY);
+            const snap = event.shiftKey ? 15 : 5;
+            this.document.updateSource({
+                direction: this.document.direction + snap * Math.sign(event.deltaY),
+                x: this.document.x,
+                y: this.document.y,
```

Comment:
Why does this need to be updated?

Reply:
Oops made an assumption about behavior that was not correct, I'll remove the x and y setting.